hangman.py

```diff
@@ -0,0 +1,250 @@
+# Problem Set 2, hangman.py
+# Name: Zlotenko Bohdana
+# Students group: KM-04
+# Collaborators: Alexey Garmash
+# Time spent: really a lot
+
+import random
+from string import ascii_lowercase
+
+WORDLIST_FILENAME = "words.txt"
+VOWELS = ('a', 'e', 'i', 'o', 'u')
```

Comment:
Tuple - is not the best container here. Try rather frozenset (it perform a quick check for letter in set, and there's no need to extend this container)